Can't find ros/ros.h when building audio_user.cpp

I gave your code a quick try on Travis-CI. `README.md` is the only file I changed before I pushed the code. Almost everything worked as expected! However, I got the following error when building `audio_user.cpp`:
```
[ 71%] Building CXX object ros-travis-integration/ros_pkg_with_dependencies/CMakeFiles/audio_user.dir/src/audio_user.cpp.o
/home/travis/catkin_ws/src/ros-travis-integration/ros_pkg_with_dependencies/src/audio_user.cpp:1:21: fatal error: ros/ros.h: No such file or directory
 #include "ros/ros.h"
                     ^
compilation terminated.
[ 71%] Built target basic_ros_pkg_generate_messages_lisp
Linking CXX shared library /home/travis/catkin_ws/devel/lib/libsquare.so
[ 85%] make[2]: *** [ros-travis-integration/ros_pkg_with_dependencies/CMakeFiles/audio_user.dir/src/audio_user.cpp.o] Error 1
make[1]: *** [ros-travis-integration/ros_pkg_with_dependencies/CMakeFiles/audio_user.dir/all] Error 2
```
You can view the entire process [here](https://travis-ci.org/kleinma/ros-travis-integration). (As of 5-5-16. I'm not sure how to get a permalink to this specific output.)

--- a/.travis.yml
+++ b/.travis.yml
@@ -111,6 +111,4 @@ before_script:
 script:
   - source /opt/ros/$ROS_DISTRO/setup.bash
   - cd ~/catkin_ws
-  - catkin_make $( [ -f $CATKIN_OPTIONS ] && cat $CATKIN_OPTIONS ) && \
-    catkin_make run_tests && \
-    catkin_make test
+  - catkin_make $( [ -f $CATKIN_OPTIONS ] && cat $CATKIN_OPTIONS ) && catkin_make run_tests && catkin_make test

--- a/.travis.yml
+++ b/.travis.yml
@@ -109,6 +109,7 @@ before_script:
 # returns non-zero when a test fails (which notifies Travis the build
 # failed). This is why we run both.
 script:
+  - source /opt/ros/$ROS_DISTRO/setup.bash
   - cd ~/catkin_ws
   - catkin_make $( [ -f $CATKIN_OPTIONS ] && cat $CATKIN_OPTIONS ) && \
     catkin_make run_tests && \

